APCAppCore/APCAppCore/Library/Categories/SBBGuidCreatedOnVersionHolder+APCAdditions.h

```diff
@@ -32,9 +32,3 @@
 // 
  
```

Comment:
This and its implementation counterpart now are also empty, to be removed entirely?